crafting-dead-core/src/main/java/com/craftingdead/core/client/renderer/item/GunRenderer.java

```diff
@@ -447,7 +449,10 @@ private void renderGun(
         ? TransformationHelper.slerp(normalTransform,
             gun.hasIronSight()
                 ? this.properties.aimTransform()
-                : this.properties.scopeAimTransform(),
+                : this.properties.scopeAimTransform().getOrDefault(gun.getAttachments().stream()

```

Comment:
This should really be cached somehow, creating a stream every frame isn't the nicest.

Reply:
Migrated the attachments to a map, removing the need for a stream